### DISCUSSION TITLE
Torrents seed, but won't upload

### DISCUSSION THREAD

**Discussion by maltschuld** [transmission/transmission#5706] [Support]
I've been using transmission for quite a while and no issues recently. I did change some settings- I'm pretty sure all I did was increase upload limits. But I must have screwed something up because all of my seeds are stuck at 0kB. Downloads work fine. What did I accidentally screw up?
Here are my settings:
"WEBPROXY_USERNAME","**************"
"WEBPROXY_PASSWORD","************"
"LOG_TO_STDOUT","false"
"HEALTH_CHECK_HOST","google.com"
"SELFHEAL","false"
"REVISION","3d97cd5302985c1a710f46ab0c311f721f224fc6"
"PIA_OPENVPN_CONFIG_BUNDLE","openvpn"
"TRANSMISSION_RPC_WHITELIST","192.168.1.148"
"TRANSMISSION_RPC_WHITELIST_ENABLED","false"
"TRANSMISSION_UPLOAD_SLOTS_PER_TORRENT","10"
"TRANSMISSION_UPLOAD_LIMIT_ENABLED","true"
"TRANSMISSION_UPLOAD_LIMIT","200"
"TRANSMISSION_START_ADDED_TORRENTS","true"
"TRANSMISSION_SPEED_LIMIT_UP_ENABLED","true"
"TRANSMISSION_SPEED_LIMIT_UP","25"
"TRANSMISSION_RATIO_LIMIT_ENABLED","false"
"TRANSMISSION_RATIO_LIMIT","2"
"TRANSMISSION_QUEUE_STALLED_MINUTES","90"
"TRANSMISSION_QUEUE_STALLED_ENABLED","true"
"TRANSMISSION_MAX_PEERS_GLOBAL","200"
"TRANSMISSION_IDLE_SEEDING_LIMIT_ENABLED","false"
"TRANSMISSION_IDLE_SEEDING_LIMIT","120"
"TRANSMISSION_DOWNLOAD_QUEUE_SIZE","12"
"TRANSMISSION_DOWNLOAD_QUEUE_ENABLED","true"
"TRANSMISSION_DOWNLOAD_LIMIT_ENABLED","false"
"TRANSMISSION_DOWNLOAD_LIMIT","100"
"TRANSMISSION_ALT_SPEED_ENABLED","false"
"TRANSMISSION_ALT_SPEED_DOWN","50"
"OVERRIDE_DNS_2","8.8.4.4"
"OVERRIDE_DNS_1","8.8.8.8"
"TRANSMISSION_RPC_PASSWORD","***************"
"TRANSMISSION_RPC_USERNAME","******************"
"TZ","us-central"
"OPENVPN_CONFIG","ca_toronto"
"LOCAL_NETWORK","192.168.1.0/24"
"OPENVPN_OPTS","--inactive 3600 --ping 10 --ping-exit 60 --mute-replay-warnings"
"TRANSMISSION_UMASK","002"
"TRANSMISSION_WATCH_DIR_ENABLED","true"

**Comment by Pentaphon** [comment]
You changed a lot of settings. I suggest going back to default and changing only the ones you need.
